test/interception.claims.test.js

```diff
@@ -191,5 +191,234 @@ addGenericTestCases((name, transferMethod, setupFunc, setupArgs = []) => {
+                await page.evaluate(() => {
+                    class MockDate extends Date {
+                        constructor(...args) {
+                            super(...args);
+                        }
+
+                        static now() {
+                            return super.now() + 60 * 60 * 1000;
+                        }
+                    }
+
+                    globalThis.__originalDate = globalThis.Date;
+                    globalThis.Date = MockDate;
+                });
+
+                await page.setRequestInterception(true);
+
+                page.on("request", req => {
+                    if (req.url() === `${BASE_URL}/refresh-custom-claim`) {
+                        customClaimRefreshCalledCount++;
+                    }
+                    req.continue();
+                });
+
+                await page.evaluate(async () => {
+                    const userId = "testing-supertokens-website";
+
+                    // Create a session
+                    const loginResponse = await toTest({
+                        url: `${BASE_URL}/login`,
+                        method: "post",
+                        headers: {
+                            Accept: "application/json",
+                            "Content-Type": "application/json"
+                        },
+                        body: JSON.stringify({ userId })
+                    });
+
+                    assertEqual(loginResponse.responseText, userId);
+
+                    const customSessionClaim = new supertokens.BooleanClaim({
+                        id: "st-custom",
+                        refresh: async () => {
+                            const resp = await toTest({
+                                url: `${BASE_URL}/refresh-custom-claim`,
+                                method: "post",
+                                headers: {
+                                    Accept: "application/json",
+                                    "Content-Type": "application/json"
+                                }
+                            });
+                        },
+                        defaultMaxAgeInSeconds: 300 /* 300 seconds */
+                    });
+
+                    const customSessionClaimValidator = customSessionClaim.validators.isTrue();
+
+                    await supertokens.validateClaims(() => [customSessionClaimValidator]);
+                    await supertokens.validateClaims(() => [customSessionClaimValidator]);
+                    await supertokens.validateClaims(() => [customSessionClaimValidator]);
+                });
+
+                assert.strictEqual(customClaimRefreshCalledCount, 1);
+            } finally {
+                await browser.close();
+            }
+        });
+    });
+});
+
+addGenericTestCases((name, transferMethod, setupFunc, setupArgs = []) => {
+    describe(`${name}: SessionClaimValidator Refresh With Clock Skew`, function () {
```

Comment:
This test case is just checking if you disable the clock skew the other test would break, right? If so we do not need this. I'm fine with you confirming that the above testcase fails if you run it with an older version of the SDK.